src/utills.js

```diff
@@ -0,0 +1,47 @@
+import { RESULT_TEXT } from "./constants.js";
+
+export const generateRandomNumber = () => {
+  return MissionUtils.Random.pickNumberInRange(0, 9);
+};
+
+export const printResultOneTurn = (cars) => {
+  const app = document.getElementById("app");
+  const resultAreaP = document.createElement("p");
+  let resultCarText = "";
+
+  for (let i = 0; i < cars.length; i += 1) {
+    resultCarText += `${cars[i].carName}: ${"-".repeat(cars[i].forwardCount)}\n`;
+  }
+
+  resultAreaP.innerText = resultCarText;
+  app.appendChild(resultAreaP);
+};
+
+export const getWinner = (cars) => {
+  let winnerList = [];
+  let largestForwardCount = 0;
+
+  for (let i = 0; i < cars.length; i += 1) {
+    if (cars[i].forwardCount > largestForwardCount) {
+      largestForwardCount = cars[i].forwardCount;
+      winnerList = [cars[i].carName];
+    } else if (cars[i].forwardCount === largestForwardCount) {
+      winnerList.push(cars[i].carName);
+    }
+  }
+
+  return winnerList;
+};
+
+export const printWinner = (winnerList) => {
+  const app = document.getElementById("app");
+  const resultSpan = document.createElement("span");
+  const winnerSpan = document.createElement("span");
+
+  winnerSpan.id = "racing-winners";
+  resultSpan.innerText = RESULT_TEXT;
+  winnerSpan.innerText = winnerList.join(", ");
+
+  app.appendChild(resultSpan);
+  app.appendChild(winnerSpan);
```

Comment:
```suggestion
  app.append(resultSpan, winnerSpan);
```

이렇게 쓸수도 있습니다 :smile:

Reply:
앗 그렇군요! 수정했습니다.

Reply:
추가 코멘트를 드리자면 `appendChild`와 `append`의 차이에 대해서도 학습해보시면 좋을 거 같아요. 단순히 인자를 몇개 받느냐도 있지만, 어떤 걸 인자로 받을 수 있는지도 다르답니다 :smile: